vertical-pod-autoscaler/pkg/recommender/input/cluster_feeder.go

```diff
@@ -49,7 +49,13 @@ import (
 	resourceclient "k8s.io/metrics/pkg/client/clientset/versioned/typed/metrics/v1beta1"
 )
 
-const defaultResyncPeriod time.Duration = 10 * time.Minute
+const (
+	scaleCacheLoopPeriod time.Duration = time.Hour
```

Comment:
I would like to differentiate between constants affecting whole cache (like the loop period) and cache entries (like refresh)

Reply:
Done.